File: apps/provider-api/Models/VwCurrentInstitutionLocationPublication.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class VwCurrentInstitutionLocationPublication
    {
        public int? InstitutionLocationPublicationId { get; set; }
        public int InstitutionId { get; set; }
        public int InstitutionLocationId { get; set; }
        public string InstitutionName { get; set; }
        public string InstitutionCode { get; set; }
        public string InstitutionStatus { get; set; }
        public string LocationName { get; set; }
        public int? LocationNumber { get; set; }
        public string LocationStatus { get; set; }
        public bool? PublishOnWebFlag { get; set; }
        public string LocationPublicationStatus { get; set; }
        public string SubRegion { get; set; }
        public string PublicationDetailStatus { get; set; }
        public Guid? LocationPublicationGuid { get; set; }
    }
}

File: libs/common/models/vw-current-institution-location-publication.ts



export class VwCurrentInstitutionLocationPublication {
    public institutionLocationPublicationId: number;
    public institutionId: number = 0;
    public institutionLocationId: number = 0;
    public institutionName: string = '';
    public institutionCode: string = '';
    public institutionStatus: string = '';
    public locationName: string = '';
    public locationNumber: number;
    public locationStatus: string = '';
    public publishOnWebFlag: boolean;
    public locationPublicationStatus: string = '';
    public subRegion: string = '';
    public publicationDetailStatus: string = '';
    public locationPublicationGuid: string;

    constructor(init? : Partial<VwCurrentInstitutionLocationPublication>) {
        
        Object.assign(this, init);
    }
}

